bin/bwallet-cli

```diff
@@ -692,12 +692,12 @@ class CLI {
         default:
           this.log('Unrecognized command.');
           this.log('Commands:');
-          this.log('  $ wallets: List all wallets.');
+          this.log('  $ backup [path]: Backup the wallet db.');
           this.log('  $ create [id]: Create wallet.');
           this.log('  $ resend: Resend pending transactions.');
-          this.log('  $ backup [path]: Backup the wallet db.');
           this.log('  $ rpc [command] [args]: Execute RPC command.'
             + ' (`bwallet-cli rpc help` for more)');
+          this.log('  $ wallets: List all wallets.');
```

Comment:
I think the `rpc` help line should stay last, maybe even follow an extra \n line break

Reply:
(same for bcoin-cli)

Reply:
I agree on `rpc` help but I think we should add an extra `\n` only [here](https://github.com/theanmolsharma/bcoin/blob/feb30a7d417bdd84260f29d2fe5d15a377ffb5f4/bin/bwallet-cli#L655-L657).

Reply:
ok